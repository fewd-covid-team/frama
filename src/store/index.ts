import { configureStore } from '@reduxjs/toolkit';
import countryReducer, {
  pickRU, pickUA, pickUSA,
} from './slice';

export default configureStore({
  reducer: {
    country: countryReducer,
  },
});

export { pickUA, pickUSA, pickRU };
