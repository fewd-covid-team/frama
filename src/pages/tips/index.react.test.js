import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import TipsScreen from './index';
import store from '../../store';

test('TipsScreen snapshot', () => {
  const component = renderer.create(
    <Provider store={store}>
      <TipsScreen />
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
