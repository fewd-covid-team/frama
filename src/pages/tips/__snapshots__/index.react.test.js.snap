// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TipsScreen snapshot 1`] = `
<div>
  <div
    className="sc-bdfBQB dYpdhA"
  >
    <div
      className="sc-gsTEea cUaASH"
    >
      <div
        className="sc-dlfnuX goJruT"
      >
        <p
          className="sc-hKgJUU delvUl"
        >
          Danil Usmanov
        </p>
        <p
          className="sc-eCstlR byKmXQ"
        >
          Wash hands!
        </p>
        <img
          className="sc-jSgvzq jiIkwK"
          src="static/emoji1.png"
        />
      </div>
    </div>
    <div
      className="sc-gsTEea kzUBZA"
    >
      <div
        className="sc-dlfnuX goJruT"
      >
        <p
          className="sc-hKgJUU delvUl"
        >
          Danil Usmanov
        </p>
        <p
          className="sc-eCstlR byKmXQ"
        >
          Wash hands!
        </p>
        <img
          className="sc-jSgvzq jiIkwK"
          src="static/emoji2.png"
        />
      </div>
    </div>
    <div
      className="sc-gsTEea iioA-Do"
    >
      <div
        className="sc-dlfnuX goJruT"
      >
        <p
          className="sc-hKgJUU delvUl"
        >
          Danil Usmanov
        </p>
        <p
          className="sc-eCstlR byKmXQ"
        >
          Wash hands!
        </p>
        <img
          className="sc-jSgvzq jiIkwK"
          src="static/emoji3.png"
        />
      </div>
    </div>
    <div
      className="sc-gsTEea iwElOC"
    >
      <div
        className="sc-dlfnuX goJruT"
      >
        <p
          className="sc-hKgJUU delvUl"
        >
          Danil Usmanov
        </p>
        <p
          className="sc-eCstlR byKmXQ"
        >
          Wash hands!
        </p>
        <img
          className="sc-jSgvzq jiIkwK"
          src="static/emoji4.png"
        />
      </div>
    </div>
    <div
      className="sc-gsTEea cIbXpN"
    >
      <div
        className="sc-dlfnuX goJruT"
      >
        <p
          className="sc-hKgJUU delvUl"
        >
          Danil Usmanov
        </p>
        <p
          className="sc-eCstlR byKmXQ"
        >
          Wash hands!
        </p>
        <img
          className="sc-jSgvzq jiIkwK"
          src="static/emoji5.png"
        />
      </div>
    </div>
  </div>
  <div
    className="sc-gKseQn idgQLy"
  >
    <img
      className="sc-iBPTik fxpbwG"
      src="static/home.png"
    />
    <p
      className="sc-fubCzh eRaIch"
    >
      Stay Home
    </p>
    <p
      className="sc-fubCzh eRaIch"
    >
      Keep Social Distance
    </p>
    <img
      className="sc-iBPTik fxpbwG"
      src="static/distance.png"
    />
    <img
      className="sc-iBPTik fxpbwG"
      src="static/hands.png"
    />
    <p
      className="sc-fubCzh eRaIch"
    >
      Wash Hands
    </p>
  </div>
</div>
`;
